crates/ruma-server-util/src/authorization.rs

```diff
@@ -39,219 +45,191 @@ impl XMatrix {
     ) -> Self {
         Self { origin, destination: Some(destination), key, sig }
     }
-}
 
-fn parse_token<'a>(tokens: &mut impl Tokens<Item = &'a u8>) -> Option<Vec<u8>> {
-    tokens.optional(|t| {
-        let token: Vec<u8> = t.take_while(|c| is_tchar(**c)).as_iter().copied().collect();
-        if !token.is_empty() {
-            Some(token)
-        } else {
-            debug!("Returning early because of empty token at {}", t.location().offset());
-            None
-        }
-    })
-}
-
-// Matrix spec:
-// > For compatibility with older servers, the recipient should allow colons to be included in
-// > values without requiring the value to be enclosed in quotes.
-fn parse_token_with_colons<'a>(tokens: &mut impl Tokens<Item = &'a u8>) -> Option<Vec<u8>> {
-    tokens.optional(|t| {
-        let token: Vec<u8> =
-            t.take_while(|c| is_tchar(**c) || **c == b':').as_iter().copied().collect();
-        if !token.is_empty() {
-            Some(token)
-        } else {
-            debug!("Returning early because of empty token at {}", t.location().offset());
-            None
-        }
-    })
-}
+    /// Parse a X-Matrix Authorization header from the given string.
```

Comment:
Seems I missed this last time
```suggestion
    /// Parse an X-Matrix Authorization header from the given string.
```

Reply:
Thanks, fixed.